iothub/service/samples/solutions/DigitalTwinClientSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -101,15 +102,16 @@ private async Task UpdateDigitalTwinComponentPropertyAsync()
                 _logger.LogDebug($"The component {Thermostat1Component} does not exist on the {_digitalTwinId} digital twin.");
 
                 // The property path to be replaced should be prepended with a '/'
-                updateOperation.AppendAddComponentOp($"/{Thermostat1Component}", componentProperty);
+                updateOperation.AppendReplace($"/{Thermostat1Component}", componentProperty);
             }
 
             _logger.LogDebug($"Update the {targetTemperaturePropertyName} property under component {Thermostat1Component} on the {_digitalTwinId} `" +
                 $"digital twin to {desiredTargetTemperature}.");
-            HttpOperationHeaderResponse<DigitalTwinUpdateHeaders> updateDigitalTwinResponse = await _digitalTwinClient
-                .UpdateDigitalTwinAsync(_digitalTwinId, updateOperation.Serialize());
+            DigitalTwinUpdateResponse updateDigitalTwinResponse = await _hubClient
+                .DigitalTwins
+                .UpdateAsync(_digitalTwinId, updateOperation.ToString());
 
-            _logger.LogDebug($"Update {_digitalTwinId} digital twin response: {updateDigitalTwinResponse.Response.StatusCode}.");
```

Comment:
@timtay-microsoft we lost the status code on this response type. Was that intended?

Reply:
This is just the HTTP status, not some application layer field of interest. We don't expose the HTTP status in other service client operations either. It is assumed that the service client method will throw if it gets a non-200 status code

Reply:
Other responses still have it, which is why I thought about it. But I realized they are direct method ones, and the status is part of the device's response.

Reply:
Yeah, the direct method case is the application-level status code, not the HTTP-level status code.